05-cond.md

```diff
@@ -65,18 +65,17 @@ which is short for "else if".
 This makes it simple to write a function that returns the sign of a number:
 
 ~~~ {.python}
-def sign(num):
-    if num > 0:
-        return 1
-    elif num == 0:
-        return 0
-    else:
-        return -1
-
-print 'sign of -3:', sign(-3)
+num = -3
+
+if num > 0:
+    print num,"is +ve"
```

Comment:
Please use "positive" - a lot of our readers will have English as a 
second language, and we shouldn't use shortcuts.

On 2015-05-04 10:48 AM, Trevor Bekolay wrote:
>
> In 05-cond.md 
> <https://github.com/swcarpentry/python-novice-inflammation/pull/77#discussion_r29590226>:
>
> > @@ -65,18 +65,17 @@ which is short for "else if".
> >  This makes it simple to write a function that returns the sign of a number:
> >
> >  ~~~ {.python}
> > -def sign(num):
> > -    if num > 0:
> > -        return 1
> > -    elif num == 0:
> > -        return 0
> > -    else:
> > -        return -1
> > -
> > -print 'sign of -3:', sign(-3)
> > +num = -3
> > +
> > +if num > 0:
> > +    print num,"is +ve"
>
> Not sure if this is just me, but I was always confused in physics 
> class when someone wrote "+ve", and would prefer the word "positive" 
> to this. But, I'm willing to concede this if most people prefer "+ve"!
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/swcarpentry/python-novice-inflammation/pull/77/files#r29590226>.
>

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org